styles/base.css

```diff
@@ -46,6 +91,13 @@ nav {
   height: auto;
   background-color: #250;
 }
+
+@media only screen and (max-width: 840px) {
```

Comment:
Good adding your media query, but I'm lost as to what's being hidden. Default should be ``min-width`` too, but I won't take off points for that, just make sure for next time.